wordpress-mobile/WordPress-Android
# Update OkHTTP library and FluxC

Fixes #https://github.com/wordpress-mobile/WordPress-FluxC-Android/issues/919

This PR is here to test the OkHTTP update of FluxC. This was previously stopped because it breaks the self-signed SSL login. I've made some changes to the way the certificate is handled and I think it's working correctly now.

Let me know if there are any more testing steps that will confirm the behaviour!

To test:
- Use fresh install of the app
- Log in with a site with a self-signed SSL certificate (use the one from FluxC tests.properties)
- Notice the dialog that let's you confirm the certificate
- Go to "Blog posts"
- Notice the posts are loaded
- Go to "Site pages"
- Notice they are loaded
- Create a post
- Publish it
- It gets published correctly

PR submission checklist:
